src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -19,6 +20,14 @@ public record RuntimeConfig
 
     public RuntimeEntities Entities { get; init; }
 
+    public DataSourceFiles? DataSourceFiles { get; init; }
```

Comment:
Please add datasourcefiles to the dab-config schema.

Reply:
added.